docker/kubernetes-tentacle/scripts/configure-and-run.sh

```diff
@@ -103,6 +106,9 @@ function validateVariables() {
     if [[ -n "$DefaultNamespace" ]]; then
         echo " - default namespace '$DefaultNamespace'"
     fi
+    if [[ -n "$TentacleCertificate" ]]; then
+        echo " - tentacle certificate custom"
```

Comment:
Could we show the first X and last X chars of the base 64 value? Something like `32B...3s=`

Reply:
A note on this one. While the update the the cert characters is a good idea, the `MII` character will appear for all X509 certs and the `==` is just empty padding characters so might not be particularly unique. 

Would it have been possible to print the thumbprint?

Reply:
It would be possible but we would have to calculate the thumbprint within the script itself and I don't think it's really worth it considering the thumbprint is logged out when the import-certificate command is run which happens a couple of lines below

```
Importing custom certificate ...
2024-05-10 01:51:53.1827 |  INFO | Importing the certificate via base64...
2024-05-10 01:51:53.2284 |  INFO | Machine key file does not yet exist. Generating key that will be used to encrypt data for this tentacle.
2024-05-10 01:51:53.2629 |  INFO | Certificate with thumbprint A21E3DAA2815C532531D354CA81887E55A8ACBFD imported successfully.
2024-05-10 01:51:53.2652 |  WARN | These changes require a restart of the Tentacle.
```

Reply:
If we are logging it below thats good, more to the point that output probably isnt particularly useful :) 
Not something to worry about at the moment since its just a log message and already merged.